python-sdk-sample/requirements.txt

```diff
@@ -1,3 +1,5 @@
 black
 isort
 zowe.core-for-zowe-sdk~=1.0.0.dev
```

Comment:
Once we publish a newer version of the core sdk, should we use that as a minimum version? 
For example: `>=1.0.0-dev16`